.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+	"presets": ["es2015"]
```

Comment:
I think `es2015` has been deprecated for a while in favor of `env`... could we try using it instead?